src/libraries/System.Private.CoreLib/src/System/Environment.cs

```diff
@@ -40,8 +40,11 @@ public readonly struct ProcessCpuUsage
         /// <summary>
         /// Gets whether the current machine has only a single processor.
         /// </summary>
+#if !FEATURE_SINGLE_THREADED
         internal static bool IsSingleProcessor => ProcessorCount == 1;
-
+#else
+        internal const bool IsSingleProcessor = true;
```

Comment:
```suggestion
        internal static bool IsSingleProcessor = true;
```

Reply:
I want this to be propagated as a constant so that pieces of code could be elided.

Reply:
would a linker substitution have accomplished the same thing?

Reply:
yes